### DISCUSSION TITLE
Putting a paragraph break inside curly brackets?

### DISCUSSION THREAD

**Discussion by puddlesofun** [inkle/ink#822] [Writing in ink]
Hi! I was wondering if there is way to break up text when it is inside curly brackets?

For instance, what I have currently:

`==FightBadDudes
{studyfighting: You expertly roundhouse kick one of the bad dudes into the other, knocking them both out, and stroll into the base. ->BaseLobby} 

{not studyfighting: You charge at one of the bad dudes, who seems unphased. He uppercuts you and you fly back several metres and land in a pile of garbage.->secretbase}`

Can result in

> You expertly roundhouse kick one of the bad dudes into the other, knocking them both out, and stroll into the base. You're in the base lobby. Ahead of you is an elevator.<

Whereas, ideally, I would like a line break before the text of ==Baselobby begins. But inkle doesn't like line breaks inside the curly brackets. Is there a workaround?

(sorry if this is obvious, I just started!)

**Comment by Selsynn** [comment]
You just have to present your curly brace with some more space, like this, to get on the multiline model. 

```
VAR studyfighting = true
==FightBadDudes
{studyfighting: 
You expertly roundhouse kick one of the bad 
dudes into the other, knocking them both out, and stroll int
o the base. ->BaseLobby
}
```

However with that model you can't have the easy straight out of the box else with the `|` but you can have this (it's the extended if-elseif-else version)

```
VAR studyfighting = true
==FightBadDudes
{
-studyfighting: 
You expertly roundhouse kick one of the bad 
dudes into the other, knocking them both out, and stroll int
o the base. 
->BaseLobby
- else: You charge at one of the bad du
des, who seems unphased. He uppercuts you 
and you fly back several metres and land in a pile of garbage.
->secretbase
}
```
